src/main/java/com/minecolonies/blocks/BlockSubstitution.java

```diff
@@ -58,8 +61,10 @@ private void initBlock()
      *
      * @return true
      */
+    //todo: remove once we no longer need to support this
+    @SuppressWarnings("deprecation")
     @Override
-    public boolean isOpaqueCube()
+    public boolean isOpaqueCube(IBlockState state)
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Don't override a deprecated method or explicitly mark it as "@Deprecated". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)